astropy/units/core.py

```diff
@@ -671,50 +669,38 @@ def _physical_type_id(self) -> tuple[tuple[str, UnitPower], ...]:
 
     @property
     def names(self) -> list[str]:
-        """
-        Returns all of the names associated with this unit.
-        """
+        """All the names associated with the unit."""
         raise AttributeError(
             "Can not get names from unnamed units. Perhaps you meant to_string()?"
         )
 
     @property
     def name(self) -> str:
-        """
-        Returns the canonical (short) name associated with this unit.
-        """
+        """The canonical (short) name associated with the unit."""
         raise AttributeError(
             "Can not get names from unnamed units. Perhaps you meant to_string()?"
         )
 
     @property
     def aliases(self) -> list[str]:
-        """
-        Returns the alias (long) names for this unit.
-        """
+        """The aliases (long) names for the unit."""
         raise AttributeError(
             "Can not get aliases from unnamed units. Perhaps you meant to_string()?"
         )
 
     @property
     def scale(self) -> UnitScale:
-        """
-        Return the scale of the unit.
-        """
+        """The scale of the unit."""
         return 1.0
 
     @property
     def bases(self) -> list[UnitBase]:
-        """
-        Return the bases of the unit.
-        """
+        """The bases of the unit."""
         return [self]
 
     @property
     def powers(self) -> list[UnitPower]:
-        """
-        Return the powers of the unit.
-        """
+        """The powers of the unit."""
```

Comment:
Maybe "The powers to which the bases of the unit are raised."

Reply:
I think this change should be made, then it LGTM!